.travis/utplsql_synonyms.sql

```diff
@@ -0,0 +1,48 @@
+whenever sqlerror exit failure rollback
+whenever oserror exit failure rollback
+set echo off
+set feedback on
+set heading off
+set verify off
+
+define ut3_owner   = "&1"
+define ut3_user    = "&2"
+define ut3_syntype = "&3"
+
+create &ut3_syntype synonym &ut3_user.ut_assert                       for &ut3_owner..ut_assert;
+create &ut3_syntype synonym &ut3_user.ut_test                       for &ut3_owner..ut_test;
+create &ut3_syntype synonym &ut3_user.ut_reporter                   for &ut3_owner..ut_reporter;
```

Comment:
Ah, now i know why you need dot in the parameter value.
That is a bit hacky though it works.